Actions/DetermineDeploymentEnvironments/DetermineDeploymentEnvironments.ps1

```diff
@@ -183,6 +187,10 @@ else {
             if ($deploymentSettings."shell" -ne 'pwsh' -and $deploymentSettings."shell" -ne 'powershell') {
                 throw "The shell setting in $settingsName must be either 'pwsh' or 'powershell'"
             }
+            if ($deploymentSettings."includeTestAppsInSandboxEnvironment" -and ($environmentName -like '* (PROD)' -or $environmentName -like '* (Production)' -or $environmentName -like '* (FAT)' -or $environmentName -like '* (Final Acceptance Test)')) {
```

Comment:
You already give a warning like this later during deployment (based on whether it actually is a sandbox environment) - I don't think i should be done here based on the naming algorithm